lib/data-fetcher.js

```diff
@@ -33,16 +36,50 @@ exports.fetchMetadataDescription = function(url) {
         reject(err);
       });
   });
-};
+}
+
+function fetchLighthouseInfo(absoluteStartUrl) {
+  return fetchJsonWithUA(LIGHTHOUSE_SERVER_URL.replace('URL', absoluteStartUrl));
+}
 
-exports.fetchLighthouseInfo = function(absoluteStartUrl) {
+function fetchJsonWithUA(url) {
   return new Promise((resolve, reject) => {
-    fetch(lighthouseServerUrl.replace('URL', absoluteStartUrl))
+    fetchWithUA(url)
       .then(response => {
         resolve(response.json());
       })
       .catch(err => {
         reject(err);
       });
   });
+}
+
+function fetchWithUA(url) {
```

Comment:
@ithinkihaveacat Yes, let me add those here. 
I had another PR for the JSDocs, but I will add another commit here.